libs/game/game.ts

```diff
@@ -139,31 +215,78 @@ namespace game {
 
     /**
      * Set the effect that occurs when the game is over
-     * @param win whether the animation should run on a win (true)
+     * @param win whether the effect should run on a win (true) or lose (false)
      * @param effect
      */
+    //% blockId=game_setgameovereffect
+    //% block="use effect $effect for $win=toggleWinLose"
+    //% effect.defl=effects.confetti
+    //% win.defl=true
+    //% group="Game Over"
+    //% weight=80
+    //% blockGap=8
+    //% help=game/set-game-over-effect
     export function setGameOverEffect(win: boolean, effect: effects.BackgroundEffect) {
         init();
-        if (!effect) return;
-        if (win)
-            winEffect = effect;
-        else
-            loseEffect = effect;
+        const goc = game.gameOverConfig();
+        goc.setEffect(win, effect, true);
     }
 
     /**
-     * Set the music that occurs when the player wins
-     * @param win
+     * Set the music that occurs when the game is over
+     * @param win whether the sound should play on a win (true) or lose (false)
      * @param effect
      */
+    //% blockId=game_setgameoversound
+    //% block="use sound $sound for $win=toggleWinLose"
```

Comment:
it's preferred to avoid setting shadows in block definitions, as that makes them harder to translate / introduces chances for translators to make mistakes (we haven't gone through and updating existing ones to avoid breaking translations for the blocks)

you can set the shadow like this https://github.com/microsoft/pxt-common-packages/blob/master/libs/game/tilemap.ts#L644 to

```typescript
//% win.shadow=toggleWinLose
```